python/run_fccanalysis.py

```diff
@@ -580,8 +608,15 @@ def run_local(config: dict[str, any],
                 'eventsProcessed',
                 nevents_orig if nevents_orig != 0 else inn)
         param.Write()
-        param = ROOT.TParameter(int)('eventsSelected', outn)
+        param = ROOT.TParameter(int)('eventsSelected', outn) # Should no of weighted, selected events be added as well? 
```

Comment:
Yes

Reply:
Added it, also to the info message it prints. Not convinced it's very pretty, happy to improve :D